### PR TITLE
Updated github flow

### PR DESCRIPTION
**Pull request by smisidjan**

### CODE REVIEW

File: Installation/Mapping/oc.githubPubliccodeComponent.mapping.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "title": "GithubPubliccodeToComponent",
   "$id": "https://api.github.com/oc.githubPubliccodeComponent.mapping.json",
   "$schema": "https://docs.commongateway.nl/schemas/Mapping.schema.json",
-  "version": "0.0.1",
+  "version": "0.0.3",
```
Comment (WilcoLouwerse) [inline Installation/Mapping/oc.githubPubliccodeComponent.mapping.json:5]: You are skipping 0.0.2 (but that isn't very important)

File: src/Service/FormInputService.php [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+     * @param CacheService                                   $cacheService           The Cache Service.
+     * @param GithubApiService                               $githubApiService       The Github Api Service.
+     * @param GithubPubliccodeService                        $publiccodeService      The Github Publiccode Service.
+     * @param ImportResourcesService                         $importResourcesService The Import Resources Service.
+     * @param EnrichPubliccodeFromGithubUrlService           $enrichPubliccode       The Enrich Publiccode From Github Url Service.
+     * @param FindGithubRepositoryThroughOrganizationService $organizationService    The find github repository through organization service.
+     * @param GatewayResourceService                         $resourceService        The Gateway Resource Service.
+     * @param LoggerInterface                                $pluginLogger           The plugin version of the logger interface
+     * @param FindOrganizationThroughRepositoriesService     $findOrganization       The find organization through repositories service.
+     */
```
Comment (WilcoLouwerse) [inline src/Service/FormInputService.php:121]: Do we need to import and construct all of these services?
Looks like a lot for such a small service

**Review by WilcoLouwerse — APPROVED** [review]

**Review by rjzondervan — APPROVED** [review]

File: src/Service/FormInputService.php (hunk 2 above)
Comment (smisidjan) [reply src/Service/FormInputService.php:121]: No I need to cleanup the imports of the services, I will do that in the next pr

File: Installation/Mapping/oc.githubPubliccodeComponent.mapping.json (hunk 1 above)
Comment (smisidjan) [reply Installation/Mapping/oc.githubPubliccodeComponent.mapping.json:5]: In te pr from Robert the version was set to 2

**Review by WilcoLouwerse — APPROVED** [review]

**Review by rjzondervan — APPROVED** [review]

**Pull request merged**